apiserver/facades/controller/singular/singular.go

```diff
@@ -53,22 +58,19 @@ type Backend interface {
 
 	// ModelTag tells the Facade what models it should consider requests for.
 	ModelTag() names.ModelTag
-
-	// SingularClaimer allows the Facade to make claims.
-	SingularClaimer() lease.Claimer
 }
 
 // NewFacade returns a singular-controller API facade, backed by the supplied
 // state, so long as the authorizer represents a controller machine.
-func NewFacade(backend Backend, auth facade.Authorizer) (*Facade, error) {
+func NewFacade(backend Backend, claimer lease.Claimer, auth facade.Authorizer) (*Facade, error) {
 	if !auth.AuthController() {
 		return nil, common.ErrPerm
 	}
 	return &Facade{
 		auth:          auth,
 		modelTag:      backend.ModelTag(),
 		controllerTag: backend.ControllerTag(),
-		claimer:       backend.SingularClaimer(),
+		claimer:       claimer,
```

Comment:
Agreed on changing it to singular claimer. 
I'm not going to change the facade attribute to LeaseClaimer though - singular and leadership are both applications of leases, so I think it's clearer this way.

Reply:
Actually, on making this change it seems a bit department of redundancy department. It's the claimer on the singular.Facade, so of course it's a singularClaimer. The uniter one made more sense since the uniter facade does so much other stuff. Ah well, I've done it now.